src/main/java/HQSE_5/ScannerStandardInput.java

```diff
@@ -0,0 +1,14 @@
+package HQSE_5;
+
+import java.util.Scanner;
+
+public class ScannerStandardInput {
+    private String UserCommand;
+
+    public String getInput() {
+        Scanner scanner = new Scanner(System.in);
```

Comment:
Is the scanner required to get reconstructed at each call ? Could it be a member ?